Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1688,10 +1712,10 @@
 
 - type: listing
   id: UplinkHardsuitSyndie
-  name: uplink-hardsuit-syndie-name
-  description: uplink-hardsuit-syndie-desc
-  icon: { sprite: /Textures/Clothing/OuterClothing/Hardsuits/syndicate.rsi, state: icon }
-  productEntity: ClothingBackpackDuffelSyndicateHardsuitBundle
+  name: Syndycate Modsuit 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)